woocommerce-gateway-ebanx/gateways/class-wc-ebanx-credit-card-gateway.php

```diff
@@ -326,15 +326,22 @@ public function process_payment($order_id)
 	public function fetch_acquirer_max_installments_for_price($price, $country = null) {
 		$max_instalments = WC_EBANX_Constants::MAX_INSTALMENTS;
 		$country = $country ?: WC()->customer->get_country();
+		$currency_code = get_woocommerce_currency();
```

Comment:
I didn't want to use strtolower on a strtoupper. I need it to be lowercase. Do you think that's ok?